### DISCUSSION TITLE
Disable links click

### DISCUSSION THREAD

**Discussion by midrissi** [Eugeny/tabby#7061] [Q&A] [answered]
Hello everyone,

Is there a way to disable the links clicks. Activate them only if we use "Ctrl+Click" for example.

It is very annoying that the terminal opens the links without prompting the user :(

Thank you for your help.

**Comment by Eugeny** [comment, marked as answer]
Yes, you can choose a modifier key in Settings -> Terminal, or disable the tabby-linkifier plugin altogether.

**Reply by midrissi** [reply]
Thank you @Eugeny for the quick reply. Actually I have found that  can just set the modifier by debugging at this line:

https://github.com/Eugeny/tabby/blob/eb1a00ab16accc66cd9f38351ed7749f0af7c809/tabby-linkifier/src/decorator.ts#L65-L66

So I have set the modifier to `altKey`

![image](https://user-images.githubusercontent.com/3237344/187096293-321de864-8565-42b7-a7d5-120e16b06429.png)
